tests/benchmark/GemmBench.cpp

```diff
@@ -88,9 +99,13 @@ class GemmBench : public Benchmark {
 
       if (param.dtype_ == ElemKind::Float16Ty) {
         bindings_.allocate(weights)->getHandle<float16_t>().clear(1.0);
+        bindings_.get(weights)->getHandle<float16_t>().randomize(
+            -1.f, 1.f, mod->getPRNG());
         bindings_.allocate(bias)->getHandle<float16_t>().clear(32);
       } else if (param.dtype_ == ElemKind::FloatTy) {
         bindings_.allocate(weights)->getHandle<float>().clear(1.0);
+        bindings_.get(weights)->getHandle<float>().randomize(-1.f, 1.f,
+                                                             mod->getPRNG());
```

Comment:
For these, we're allocating, clearing to 1, and then randomizing. Could instead: 
```suggestion
        bindings_.allocate(weights)->getHandle<float16_t>().randomize(-1.f, 1.f, mod->getPRNG());
        bindings_.allocate(bias)->getHandle<float16_t>().clear(32);
      } else if (param.dtype_ == ElemKind::FloatTy) {
        bindings_.allocate(weights)->getHandle<float>().randomize(-1.f, 1.f, mod->getPRNG());
```

Reply:
Will do